### DISCUSSION TITLE
Planes on offline map

### DISCUSSION THREAD

**Discussion by it9ybg** [szpajder/dumphfdl#37] [Q&A] [answered]
Hi, I installed DumpHFDL on my RaspberryPi3, everything went fine, it works with VRS (Via TCP and VRS with Windows). I would like to leave ONLY my raspberryPi on, for example 2 days and then see, using the DumpHFDL .log file, all the planes received in 2 days, in a map like Google Earth or something else, is this possible? Thanks for any ideas.

**Comment by szpajder** [comment, marked as answer]
Yes, it is possible.

- Enable logging of aircraft positions to a separate file in a Basestation format (`--output decoded:basestation:file=/some_dir/positions.log,rotate=daily`).

- Extract required fields from the Basestation file - latitude and longitude is required as a minimum, everything else is optional. For example, in order to create a new file named `positions.csv` containing only coordinates, use `cut -d, -f15,16 /some_dir/positions.log > positions.csv`. 15 and 16 are field numbers to be extracted, numbering starts from 1.

- Open the newly created file with a text editor and add a header row at the top with field names. Save the file. It should look more or less like this:

```
lat,lon
32.043866,15.128502
17.123456,-100.363296
... etc.
```

Use your favorite GIS app to plot the resulting file on the map. For example in QGIS:

- start a new empty project (or open an existing one)

- from the top menu select Layer -> Add Layer -> Add Delimited Text Layer

- type the path to the positions.csv file in the "File name" field (or press the "..." button to browse and select it)

- inspect the "Sample data" section at the bottom and check if the fields names and their contents have been correctly recognized 

- click "Add".

You should see the position points plotted in the main window. Obviously you need to add some map as a background, but this is off topic here - there is plenty of tutorials about QGIS around the web.

**Comment by it9ybg** [comment]
Thanks a lot, great idea. I use Google maps directly. 
[https://it9ybg.blogspot.com/2023/03/dumphfdl-decoder.html](url)
